src/openrct2/network/network.h

```diff
@@ -20,10 +20,10 @@
 #include <string_view>
 #include <vector>
 
-constexpr int32_t kNetworkDefaultPort = 11753;
+constexpr uint16_t kNetworkDefaultPort = 11753;
```

Comment:
Not sure why you changed it back to an `unsigned`, it looks wrong since all assignments on https://github.com/OpenRCT2/OpenRCT2/pull/21705/commits/e167949ca9e41849622fadf20c6ec95cd37dc52b are done to `int32_t` and there are even functions to read from/to an `int32_t`

Reply:
Fix in #21760